File: BlueSkyTaskThree - java version/src/main/java/com/bluesky/model/destination/UserGroup.java
package com.bluesky.model.destination;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;


@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class UserGroup {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private String id;
    private String uid;
    private String gid;


}

File: BlueSkyTaskThree - kotlin version/src/main/kotlin/com/bluesky/model/destination/UserGroup.kt
package com.bluesky.model.destination
import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
class UserGroup {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val id:String = ""
    private val uid:String = ""
    private val gid:String = ""
}